### PR TITLE
fix toast message on widget cut

### LINKED ISSUE #983: [Bug] Inappropriate message is displayed to user on CUT functionality
## Steps to reproduce

1. Add a widget
2. Using Keypad CMD+X (CUT) the widget

and observe the message 

## Observation

It is observed that the user is displayed with message "Copied Tabs1" when the widget is CUT

## Expected behaviour

User must be displayed with appropriate message stating the widget is cut

### PR DESCRIPTION
**Pull request by rishabhsaxena**
## Description
- Common function to create a widget copy
- Trigger separate toast/events for cut 

Fixes # (issue)
- fixes #983 

## Type of change
- Bug fix

## How Has This Been Tested?
- both copy and cut shortcuts manually

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/client/src/sagas/WidgetOperationSagas.tsx [hunk 1]
```diff
@@ -810,21 +810,32 @@ function* updateCanvasSize(
   }
 }
 
-function* copyWidgetSaga(action: ReduxAction<{ isShortcut: boolean }>) {
+function* createWidgetCopy() {
   const selectedWidget = yield select(getSelectedWidget);
   if (!selectedWidget) return;
```
Comment (hetunandu) [inline app/client/src/sagas/WidgetOperationSagas.tsx:815]: Can't we show some message saying nothing is selected?

File: app/client/src/sagas/WidgetOperationSagas.tsx [hunk 2]
```diff
@@ -1068,12 +1079,24 @@ function* pasteWidgetSaga() {
 }
 
 function* cutWidgetSaga() {
-  yield put({
-    type: ReduxActionTypes.COPY_SELECTED_WIDGET_INIT,
-    payload: {
-      isShortcut: true, // We only have shortcut based "cut" operation today.
-    },
+  const selectedWidget = yield select(getSelectedWidget);
+  if (!selectedWidget) return;
```
Comment (hetunandu) [inline app/client/src/sagas/WidgetOperationSagas.tsx:1083]: Don't error out silently. Show some message if nothing is selected

**Review by hetunandu — CHANGES_REQUESTED** [review]
UX improvements needed

**Comment by rishabhsaxena** [comment]
@hetunandu have updated at the root of the copy/cut sagas

**Review by hetunandu — APPROVED** [review]

**Pull request merged**